fixieai/agents-ts/package.json

```diff
@@ -1,20 +1,22 @@
 {
-  "name": "@fixie/sdk",
-  "private": true,
-  "version": "1.0.0",
+  "name": "@fixieai/sdk",
+  "version": "1.0.0-6",
```

Comment:
Unfortunately, I don't see a better way to dev/test these PRs without publishing a bunch of prerelease versions. Any way we work around this would be more noise than it's worth. The prerelease versions are a little annoying but I think almost no one will even notice.